plugins/trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/get_rule_details/action.py

```diff
@@ -30,7 +31,7 @@ def run(self, params={}):
         url = f"{self.connection.dsm_url}/api/intrusionpreventionrules/{self.id}"
 
         # Send request
-        response = self.connection.session.get(url, verify=self.connection.dsm_verify_ssl)
+        response = requests.post(url, verify=self.connection.dsm_verify_ssl, headers=self.connection.headers)
```

Comment:
Is this a copy-paste error? I think this should be a GET request:

https://automation.deepsecurity.trendmicro.com/article/12_0/api-reference/tag/Intrusion-Prevention-Rules#operation/describeIntrusionPreventionRule

Reply:
Yep!